licenses/find.go

```diff
@@ -42,6 +44,7 @@ func Find(dir string, classifier Classifier) (string, error) {
 	return findUpwards(dir, licenseRegexp, stopAt, func(path string) bool {
 		// TODO(RJPercival): Return license details
 		if _, _, err := classifier.Identify(path); err != nil {
+			glog.Warningf("classifier.Identify(%s): %v", path, err)
```

Comment:
This is probably better as Info() logging, since it's rarely indicative of a problem unless you're writing tests and forget to update the classifier stub.

Reply:
Done.